buildSrc/src/main/kotlin/Project.kt

```diff
@@ -85,7 +85,7 @@ object Versions {
   const val mapboxJavaServices = "5.4.1"
   const val mapboxBase = "0.5.0"
   const val mapboxGlNative = "10.4.2"
-  const val mapboxCommon = "21.2.0"
+  const val mapboxCommon = "21.2.1"
```

Comment:
bumping common is enough, correct? I mean, no gl-native bump needed

Reply:
yeah no gl-native bump is needed.